src/client/core.ts

```diff
@@ -88,7 +83,7 @@ export async function aptosRequest<Req, Res>(
       throw new AptosApiError(
         options,
         result,
-        response.data.errors[0].message ?? `Unhandled Error ${response.status} : ${response.statusText}`,
+        (response.data as any).errors[0].message ?? `Unhandled Error ${response.status} : ${response.statusText}`,
```

Comment:
Interesting, why `any`?

Reply:
before, `request` returned `Promise<any>` and therefore ts will resolve to any type and you can pretty much do whatever you want. now, we return a `ClientResponse<Res>` type where the `data` in `ClientResponse` is `Res`. Indexer is a bit weird in a way that errors are part of the data key returned by the api, so since we dont really want to do `type ClientResponse = {.... data:Res & {errors?:any} } }` we just convert it here to `any`

Reply:
btw, we also do it in the `if` block